packet/packet.py

```diff
@@ -1,4 +1,4 @@
-from .models import Freshman, UpperSignature, FreshSignature, MiscSignature, db
+from .models import Freshman, UpperSignature, FreshSignature, MiscSignature, db, datetime
```

Comment:
should datetime be a separate import rather than relying on it being imported in `models.py`?

```python
from datetime import datetime
```

Reply:
Yes, yes it should.  Pretty sure I alt-entered that, but i don't remember